jsonschema-validation.xml

```diff
@@ -536,84 +530,65 @@
                     <cref>
                         Note that the "type" keyword in this specification defines an "integer" type
-                        which is not part of the data model.  Therefore a format attribute can be
-                        limited to numbers, but not specifically to integers.  However, a numeric
+                        which is not part of the data model. Therefore a format attribute can be
+                        limited to numbers, but not specifically to integers. However, a numeric
                         format can be used alongside the "type" keyword with a value of "integer",
                         or could be explicitly defined to always pass if the number is not an integer,
                         which produces essentially the same behavior as only applying to integers.
                     </cref>
                 </t>
 
                 <t>
-                    Meta-schemas that do not use "$vocabulary" SHOULD be considered to
-                    utilize this vocabulary as if its URI were present with a value of false.
-                    See the Implementation Requirements below for details.
+                    The current URI for this vocabulary, known as the Format-Annotation vocabulary, is:
+                    &lt;https://json-schema.org/draft/2020-11/vocab/format-annotation&gt;. The current
+                    URI for the corresponding meta-schema is:
+                    <eref target="https://json-schema.org/draft/2020-11/meta/format-annotation"/>.
+                    Implementing support for this vocabulary is REQUIRED.
                 </t>
                 <t>
-                    The current URI for this vocabulary, known as the Format vocabulary, is:
-                    &lt;https://json-schema.org/draft/2020-11/vocab/format&gt;.
+                    In addition to the Format-Annotation vocabulary, a secondary vocabulary is available
+                    for custom meta-schemas that defines "format" as an assertion. The URI for the
+                    Format-Assertion vocabulary, is:
+                    &lt;https://json-schema.org/draft/2020-11/vocab/format-assertion&gt;. The current
+                    URI for the corresponding meta-schema is:
+                    <eref target="https://json-schema.org/draft/2020-11/meta/format-assertion"/>.
+                    Implementing support for the Format-Assertion vocabulary is OPTIONAL.
                 </t>
                 <t>
-                    The current URI for the corresponding meta-schema is:
-                    <eref target="https://json-schema.org/draft/2020-11/meta/format"/>.
+                    Specifying both the Format-Annotation and the Format-Assertion vocabularies is functionally
+                    equivalent to specifying only the Format-Assertion vocabulary since its requirements
+                    are a superset of the Format-Annotation vocabulary.
```

Comment:
This isn't really true -- if format validation fails, the result doesn't contain an annotation.  If one wants an annotation in *all* cases, then the user can specify both vocabularies.

Reply:
> if format validation fails, the result doesn't contain an annotation

If the format validation fails, then its parent schema will fail, and no annotations will be returned anyway.

Reply:
ok fair.